content/02.introduction.md

```diff
@@ -24,69 +24,59 @@ Further, we aim to amplify the true signal out of the vast noise produced by tho
+Through innate immunity, the body can rapidly recognize and attack pathogens and infected cells. 
+Through the innate immune response, cells can also pass along critical information to prime the adaptive immune response.
+In this phase, the immune response recognizes and reacts to a set of signatures broadly associated with the presence of foreign particles that are likely to be pathogenic, e.g., double-stranded RNA (dsRNA) is never produced endogenously and thus its presence indicates viral replication is likely occurring. 
+The adaptive immune response targets specific protein sequences associated with specific pathogens. 
+This branch of the immune response can be broadly classified into the humoral and cell-mediated responses. 
+The humoral response includes soluble proteins and antibodies that can neutralize and eliminate pathogens in circulation that have not yet entered cells and can also mediate killing of pathogen-infected cells. 
+Cell-mediated responses include the killing of pathogen-infected cells by cytotoxic CD8+ T cells and the secretion of cytokines to drive inflammation and support immune cell activities.
+Thus, pathogens infects hosts and hosts have immune defenses and responses; furthermore, pathogens have developed ways to evade the immune system (as well as, in general, various ways to manipulate the host).
+Other general complexities to consider include that disease symptoms can be due to a pathogen or due to the reaction of the immune system to the pathogen and that the host immune system can defend against or fight infection but it can also sometimes cause too much damage to the host [@doi: 978-0815332183; @doi: 9780071283663].
+
+##### SARS-CoV-2 and the Immune Response
+
+The first line of defense for the immune system is physical barriers such as the skin and mucus, but viruses such as SARS-CoV that infect the epithelium are able to bypass these defenses [@doi:10.1038/mi.2015.127].
+The innate immune response targets pathogens in circulation.
+This response depends on several types of leukocytes (white blood cells) that are professional phagocytic cells, which means they are able to detect and engulf an antigen, enveloping it in a phagosome.
+The conditions within the phagosome are then modulated to destroy the pathogen through fusion of the phagosome with a lysosome or granule to form a phagolysosome.
+This allows the cell to break the pathogen into its component peptides. 
+Monocytes, which include both dendritic cells and macrophages, are both phagocytic white blood cells and professional antigen-presenting cells, which meaning that they can then present the resulting viral peptides on the cell surface using Major Histocompatability Complex II (MHC-II) proteins.
+However, research in related viruses suggests that SARS-CoV-2 may possess some mechanisms that facilitate evasion of the antibody-mediated response.
```

Comment:
I'm not sure how this sentence relates to what's around it (which may be because I haven't read the articles you are referring to -- but still it's a bit confusing to read because the mechanisms it mentions seem different than the previous and following sentence). This is just a note/reminder to sort this out.